doc/2/protocols/http/constructor/index.md

```diff
@@ -13,14 +13,15 @@ Use this constructor to create a new instance of the `Http` protocol with specif
 ## Arguments
 
 ```dart
-HttpProtocol(Uri);
+HttpProtocol(Uri, {bool acceptUnsignedCertificate = false});
```

Comment:
```suggestion
HttpProtocol(Uri, {bool acceptBadCertificate = false});
```

Reply:
Should be more generic since a certificate can be invalid because of expiry date